db/migrations/20170418171151_create_movies_locations.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+exports.up = function (Knex, Promise) {
+  return Knex.schema.createTable('movies_locations', (table) => {
+    table.increments('id').primary();
+    table.integer('loc_id');
```

Comment:
Would you also have to do something like in line 5 where it says .primary(), and do like .foreign() and like specifically reference another table by its name??